src/style.css

```diff
@@ -0,0 +1,26 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+  height: 16px;
```

Comment:
```suggestion
  align-items: center;
```
![image](https://github.com/mate-academy/layout_stars/assets/96409563/205e336f-01b8-4180-a77c-e4400ba926bf)